TAF_EasyRest/TestFramework/Pages/BasePage.cs

```diff
@@ -28,5 +28,19 @@ public virtual void GoToUrl()
         {
             driver.Navigate().GoToUrl(baseUrl);
         }
+
+        protected IWebElement WaitUntilElementIsVisible(By locator, int timeToWait)
```

Comment:
I think it is better to make this method an extension method of IWebDriver to avoid code duplication in the BasePage and BasePageComponent classes. What do you think?

Reply:
agree with you, it`s a good idea

Reply:
done